src/containers/CartPage.less

```diff
@@ -0,0 +1,275 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+ @import './../style/common.less';
+
+.cart-container{
+  min-height: 100%;
+  margin-bottom: 0px;
+  :after {
+    content: "";
+    display: block;
+  }
+  
+  @media screen and (max-width: 768px) {
+    table {
+      display: block;
+      width: 100%;
+    }
+
+    table thead {
+      display: block;
+      width: 100%;
+    }
+
+    table tbody {
+      display: block;
+      width: 100%;
+    }
+    table tbody tr {
+      display: block;
+      width: 100%;
+    }
+    table tbody tr td {
+      display: block!important;
+      width: 100%!important;
```

Comment:
Same as previous, avoid `!important`, if you use it put a space before it, if it looks ok the way it's written right now, try to remove the whole line. If you can remove this and previous line, then remove the whole `td` part.